### DISCUSSION TITLE
Issue cards in Project boards cannot be reordered in the same lane

### DISCUSSION THREAD

**Discussion by WasiqB** [community/community#46128] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

Earlier I was able to rearrange issues in the projects board view in the same lane by putting high priority issues on the top of the lane.

But recently all the cards are displayed in ascending order based on the ticket number. I am no longer move the cards around in the same lane based on the priority.

**Comment by JazielGuerrero** [comment]
In my case I had specified a sorting configuration globally in the board, this prevented me from sorting individuals card in the lane/column. I put it back to **no sorting**, and then I could once again rearrange the cards individually. 

![Screen Shot 2023-02-26 at 10 32 22 AM](https://user-images.githubusercontent.com/27296925/221416957-e6feb231-790c-400f-9528-8e07fab24159.png)

**Comment by rileybroughten** [comment, marked as answer]
👋 This is expected behavior as when you are sorting by a field, you can still manually move items _between_ columns with the sorted order maintained, just not manually reordering them _within_ the column since it is sorted. Here is the [documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/customizing-the-board-layout#sorting-by-field-values) for more details!
